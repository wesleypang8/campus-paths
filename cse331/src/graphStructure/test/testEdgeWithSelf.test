#test adding an edge to self

#create graph
CreateGraph graph1

#add node
AddNode graph1 node1

#add edge
AddEdge graph1 node1 node1 selfEdge

#print children/ edge
ListChildren graph1 node1