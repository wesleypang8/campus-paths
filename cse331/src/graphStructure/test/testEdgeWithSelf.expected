#test adding an edge to self

#create graph
created graph graph1

#add node
added node node1 to graph1

#add edge
added edge selfEdge from node1 to node1 in graph1

#print children/ edge
the children of node1 in graph1 are: node1(selfEdge)